swift-style-guide.md

```diff
@@ -219,3 +219,56 @@ _Rationale:_ Omitting redundant type parameters clarifies the intent, and makes
 
 #### Curly braces go on the same line
 In every situation.
+
+## Code Organization
+
+Use extensions to organize your code into logical blocks of functionality. Each extension should be set off with a `// MARK: -` comment to keep things well-organized.
```

Comment:
Add a blurb about how these marks show up in the 'show document items' menu (accessible via `⌃6`)